src/LICM.cpp

```diff
@@ -89,6 +89,10 @@ class LiftLoopInvariants : public IRMutator {
                 return false;
             }
         }
+        if (const Reinterpret *reinterpret = e.as<Reinterpret>()) {
+            // Don't lift these intrinsics. They're free.
```

Comment:
Not an intrinsic, so maybe alter comment to say "Reinterpret nodes" instead of "these intrinsics"

Reply:
Whoops, i forgot.

Reply:
Ugh, i hate github UI.